<?xml version="1.0" ?>

<launch>

  <arg name="robot_ip" default="130.230.37.115" />
  <arg name="load_gripper" default="true" />

  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <include file="$(find cobot_controllers)/launch/moveit_control.launch">
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <node name="move_to_start" pkg="cobot_controllers" type="move_to_start.py" respawn="false" output="screen"/>


</launch>
